### ISSUE TITLE
Evaluation script

### ISSUE THREAD

**Issue by MistreanuIonutCosmin** [microsoft/CoCosNet#19]
Hello! 
Can you point me out to a reference implementation for the FID and SFW metrics? I've tried to reproduce the results, but I couldn't achieve the same numbers as you present in the paper. I don't know where is the difference coming from. Thank you!

**Comment by fnzhan** [comment]
Similar problem, I trained the model on Celeba-HQ with default setting and cannot reproduce the FID=14.3.  Hi @panzhang0212, how can we reproduce the score reported in the paper?

**Comment by fnzhan** [comment]
I evaluate on Celeba-HQ with the provided pre-trained model , and obtain a fid of 16.95. If anyone reach the reported fid of 14.3?

**Comment by panzhang0212** [comment]
![image](https://user-images.githubusercontent.com/30363822/95820766-4eca5500-0d5b-11eb-92a1-9bfcba2f1a86.png)
I run the test code again, The FID is 14.3. the test code is repo https://github.com/shepnerd/pytorch-fid, you can also try https://github.com/mseitzer/pytorch-fid. All test images and ground truth are [here](https://drive.google.com/drive/folders/1RPAoxyuQW9gul0RWNrbNoJXn1apenz3V?usp=sharing).

**Comment by fnzhan** [comment]
> ![image](https://user-images.githubusercontent.com/30363822/95820766-4eca5500-0d5b-11eb-92a1-9bfcba2f1a86.png)
> I run the test code again, The FID is 14.3. the test code is repo https://github.com/shepnerd/pytorch-fid, you can also try https://github.com/mseitzer/pytorch-fid. All test images and ground truth are [here](https://drive.google.com/drive/folders/1RPAoxyuQW9gul0RWNrbNoJXn1apenz3V?usp=sharing).

Thanks for your reply, I check the results again, and find that saving the results as '.png' format will decrease the FID, saving as '.jpg' can reproduce the reported FID.

**Comment by fnzhan** [comment]
> ![image](https://user-images.githubusercontent.com/30363822/95820766-4eca5500-0d5b-11eb-92a1-9bfcba2f1a86.png)
> I run the test code again, The FID is 14.3. the test code is repo https://github.com/shepnerd/pytorch-fid, you can also try https://github.com/mseitzer/pytorch-fid. All test images and ground truth are [here](https://drive.google.com/drive/folders/1RPAoxyuQW9gul0RWNrbNoJXn1apenz3V?usp=sharing).

Hi, @panzhang0212 , can you give a link to the repo of the SWD metric used in the paper? Thanks.

**Comment by panzhang0212** [comment]
Please refer to #25
